lib/main/atom/commands/goToDeclaration.ts

```diff
@@ -2,6 +2,15 @@ import {commands} from "./registry"
 import {commandForTypeScript, getFilePathPosition} from "../utils"
 import {simpleSelectionView} from "../views/simpleSelectionView"
 
+const prevCursorPositions:any[] = [];
```

Comment:
Missing the type annotation here. Once you add it, you'll discover a type mismatch between what `getFilePathPosition` returns and what `open` expects.